modules/cnv-deleting-hco-subscription.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_install/uninstalling-container-native-virtualization.adoc
+
+[id="cnv-deleting-hco-subscription_{context}"]
+= Deleting the KubeVirt HyperConverged Cluster Operator catalog subscription
+
+To finish uninstalling {ProductName}, delete your KubeVirt HyperConverged
+Cluster Operator catalog subscription.
+
+.Prerequisites
+
+* An active *KubeVirt HyperConverged Cluster Operator* catalog subscription
+
+.Procedure
+
+. Ensure that you are in the `kubevirt-hyperconverged` project. If not, select
+it from the *Project* list in the {product-title} web console.
+
+. Navigate to the *Catalog* -> *Operator Management* page.
+
+. Click the *Operator Subscriptions* tab.
+
+. Click the Options menu in the row containing the *hco-subscription*
+subscription. In the expanded menu, click *Remove Subscription*.
+
+. Confirm that you want to remove the subscription by clicking *Remove* again.
```

Comment:
You might be able to fold this step into the previous one. You might also want to add a quick way for a user to confirm that it's gone, even if it's "The \<thing> is removed from the \<list\>."